### ISSUE TITLE
ReferenceError: btoa is not defined

### ISSUE THREAD

**Issue by davidtperk** [MyEtherWallet/MEWconnect-web-client#39]
Seeing the following error using @myetherwallet/mewconnect-web-client version 2.1.22.

```
ReferenceError: btoa is not defined
    at Object.<anonymous> (my_project/node_modules/@myetherwallet/mewconnect-web-client/dist/index.js:4207:1)
```

Is this the latest production build? Thanks.

**Comment by georgeroman** [comment]
+ 1

**Comment by vfn** [comment]
I had this issue when using it with Next.JS, and I believe this would happen with any other server-side-rendering approach.

The library assumes it will be running on a browser, I managed to work around this by lazy loading it after DOM is ready. For example, I loaded it inside `componentDidMount`.

```js
let MewConnect = null;
class MyComponent extends React.Component {
  async componentDidMount() {
    const { default: MewConnectImport } = await import('@myetherwallet/mewconnect-web-client');
    MewConnect = MewConnectImport;
    ...
    // Proceed with using `MewConnect`
  }
...
}
```

**Comment by kvhnuke** [comment]
should be fixed in a68bcc0f
